src/momento/typing.py

```diff
@@ -14,8 +14,11 @@
 # List Types
 TListName = TCollectionName
 TListValue = Union[str, bytes]
-TListValuesBytes = Sequence[bytes]
-TListValuesStr = Sequence[str]
-TListValues = Union[TListValuesBytes, TListValuesStr]
+TListValuesInputBytes = Iterator[bytes]
+TListValuesInputStr = Iterator[str]
+TListValuesInput = Union[TListValuesInputBytes, TListValuesInputStr]
```

Comment:
Inputs should be `Iterable[TListValue]`, right? Ie support heterogeneous inputs

Reply:
Just pushed a fix.